autorandr.py

```diff
@@ -715,6 +716,20 @@ def generate_virtual_profile(configuration, modes, profile_name):
                 shift += int(mode[shift_index])
             else:
                 configuration[output].options["off"] = None
+    elif profile_name == "clone":
+        biggest_width_resolution = sorted([output_modes[0] for output, output_modes in modes.items()], key=lambda x: x["width"], reverse=True)[0]
```

Comment:
There are people using screens in vertical mode (9:16). How about sorting by `width*height` instead?